api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -1447,22 +1467,38 @@ def tip_racks(self, racks: List[labware.Labware]) -> None:
 
     @property  # type: ignore
     @requires_version(2, 0)
-    def trash_container(self) -> labware.Labware:
-        """The trash container associated with this pipette.
+    def trash_container(
+        self
+    ) -> Union[labware.Labware, TrashBin, WasteChute]:
+        """The trash containers associated with this pipette.
 
         This is the property used to determine where to drop tips and blow out liquids
         when calling :py:meth:`drop_tip` or :py:meth:`blow_out` without arguments.
 
-        By default, the trash container is in slot A3 on Flex and in slot 12 on OT-2.
+        On a Flex running a protocol with API Version >= 2.16, `trash_container` returns
+        the first TrashBin or WateChute object loaded in the protocol.
+        On a Flex running a protocol with API Version < 2.16, `trash_container` returns
+        a single-well fixed trash labware in slot D3.
+        On a an OT-2, `trash_container` always returns a single-well fixed trash labware
+        in slot 12.
+
+        .. versionchanged:: 2.16
+            Adds the support for TrashBin and WasteChute objects.
```

Comment:
Great! This suggestion is mostly formatting cleanup. (RST double backticks, sigh…) Also clarifying that it doesn't return anything because it's a property, not a method.

```suggestion
        On a Flex running a protocol with API version 2.16 or higher, ``trash_container`` is
        the first ``TrashBin`` or ``WasteChute`` object loaded in the protocol.
        On a Flex running a protocol with API version 2.15, ``trash_container`` is
        a single-well fixed trash labware in slot D3.
        On a an OT-2, ``trash_container`` is always a single-well fixed trash labware
        in slot 12.

        .. versionchanged:: 2.16
            Added support for ``TrashBin`` and ``WasteChute`` objects.
```